docs/resources/project_access.md

```diff
@@ -13,6 +13,11 @@ ProjectAccess schema
 ## Example Usage
 
 ```terraform
+import {
+  project = "sample"
+  to      = unleash_project_access.sample_project_access
+}
```

Comment:
Not sure if this will work, because the resource is being created in line 50. What you can import maybe is the default project access

Reply:
Got it thanks. Importing the default project access and overriding the owner (I think)